docs/faq.md

```diff
@@ -230,112 +230,110 @@ With the release of version 6.0, we updated our documentation as well. That does
+|                  | `main.js` configuration/webpackFinal            | [See current documentation](./api/main-config-webpack-final.md)                   | Non existing feature or undocumented                                                                                                                                                                                                                                 | Non existing feature or undocumented                                                                                                                     |
+|                  | Frameworks                                      | [See current documentation](./api/new-frameworks.md)                              | Non existing feature or undocumented                                                                                                                                                                                                                                 | Non existing feature or undocumented                                                                                                                     |
+|                  | CLI options                                     | [See current documentation](./api/cli-options.md)                                 | [See versioned documentation](https://github.com/storybookjs/storybook/tree/release/5.3/docs/src/pages/configurations/cli-options)                                                                                                                                   | [See versioned documentation](https://github.com/storybookjs/storybook/tree/release/5.0/docs/src/pages/configurations/cli-options)                       |
+
 <Callout variant="info">
 
-With the release of version 5.3, we've updated how you can write your stories more compactly and easily. It doesn't mean that the `storiesOf` format has been removed. For the time being, we're still supporting it, and we have documentation for it. But be advised that this is bound to change in the future.
+If you have stories written with the older `storiesOf` format, it was removed in Storybook 8.0 and is no longer maintained. We recommend that you migrate your stories to CSF. See the [migration guide](./migration-guide.md#storiesof-to-csf) for more information. However, if you need, you can still access the old documentation for `storiesOf` [here](https://github.com/storybookjs/storybook/blob/release/5.3/docs/src/pages/formats/storiesof-api/index.md).
```

Comment:
Minor: Let's avoid links that use only "here" for their text:

```suggestion
If you have stories written with the older `storiesOf` format, it was removed in Storybook 8.0 and is no longer maintained. We recommend that you migrate your stories to CSF. See the [migration guide](./migration-guide.md#storiesof-to-csf) for more information. However, if you need, you can still access the [old documentation for `storiesOf`](https://github.com/storybookjs/storybook/blob/release/5.3/docs/src/pages/formats/storiesof-api/index.md).
```

Reply:
I fail to see your point on this. We have it across the documentation, and it's not incorrect. Let me know and we'll go from there.

Reply:
First, I don't think it's a blocker. Second, it's an accessibility thing. Users of assistive technology (like screen readers) can pull up a list of all the links on a page. When they do so, they need the link text to be descriptive, so they can know what to expect when they activate that link. That's especially true when the URL is ungainly as it is here (nothing we can do about that). When the link text is just "here", that provides zero information about what that link is pointing to.